src/mono/HowToUpgradeMono.md

```diff
@@ -80,15 +71,13 @@ Currently the build reports these warnings:
 
  * `EXEC : warning : unresolved symbol: putchar` (during both wasm and asm.js builds)
  * `EXEC : warning : root:BINARYEN_ASYNC_COMPILATION disabled due to user options` (during the asm.js build only)
- * `EXEC : warning : unresolved symbol: mono_wasm_invoke_js_with_args` (during both wasm and asm.js builds). I expect this is a temporary issue that will go away when Mono finishes implementing whatever they are doing with their JS interop, which we aren't currently using anyway since we have our own interop in `driver_blazor.c`.
- * `EXEC : warning : unresolved symbol: mono_sdb_single_step_trampoline` (during both wasm and asm.js builds). I expect this will eventually be implemented as part of the debugging work.
+ * `EXEC : warning : unresolved symbol: mono_wasm_invoke_js_with_args / mono_wasm_get_object_property / mono_wasm_set_object_property` (during both wasm and asm.js builds). I expect this is due to us not including Mono's `binding_support.js` in the build, which we could do but don't presently require.
```

Comment:
@SteveSandersonMS,   Yes these are due to not including the `binding_support.js`.